src/Microsoft.AspNetCore.Authentication.Twitter/TwitterHandler.cs

```diff
@@ -161,7 +161,7 @@ public TwitterHandler(IOptionsSnapshot<TwitterOptions> options, ILoggerFactory l
             var cookieOptions = new CookieOptions
             {
                 HttpOnly = true,
-                SameSite = SameSiteMode.Lax,
+                SameSite = SameSiteMode.Strict,
```

Comment:
Same remark as above. Twitter - that implements OAuth1 - uses GET/302 responses to return the user back to the client app. If `Strict` doesn't work with the final application cookie due to the cross-origin 302 dance - as explained by @anurse - it will likely not work here.